src/tests/DataMapperTests.cpp

```diff
@@ -182,7 +182,7 @@ TEST_CASE_METHOD(SqlTestFixture, "iterate over database", "[SqlRowIterator]")
     for (int i = 40; i <= 50; ++i)
     {
         auto person = Person {};
-        person.name = "John";
+        person.name = SqlAnsiString<25>(std::format("John-{}", i));
```

Comment:
```suggestion
        person.name = std::format("John-{}", i);
```
Looks like we rather want to add a convenience assignment operator to allow assigning an `std::basic_string<T>` to an `SqlFixedString<N, T>`, leaving that line change as intuitive as suggested above. :)

Reply:
this looks like not a very trivial change since person.name here is a field and we need to properly propagate constructors, and test "SqlFixedString: assign" already creates some API that we need to preserve

Reply:
Rebase on top of hopefully soon merged #202, and then you should be good to go?

Reply:
this is the only thing missing now, right?

Reply:
Yes